code/modules/projectiles/bullet.dm

```diff
@@ -263,6 +263,10 @@ toxic - poisons
 	name = "stun bullet"
 	power = 20
 	ks_ratio = 0.0
+//How fast the power goes away
+	dissipation_rate = 5
+//How many tiles till it starts to lose power
```

Comment:
comments should be after the vars here to be cleaner

Reply:
done